README.md

```diff
@@ -1,24 +1,71 @@
-# README
+# Budgetplus
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+> |Homepage|Details page|
+> |-------------------------------------|-------------------------------------|
+> |![screen1](https://user-images.githubusercontent.com/9049260/154673640-28a66eff-be3e-4e85-93b7-60b1c9b3f747.png) | ![screen2](https://user-images.githubusercontent.com/9049260/154673687-4180322b-3da4-4b64-822b-533870b56c1b.png)|!
 
-* Ruby version
+## Getting Started
 
-* System dependencies
+To get a local copy up and running follow these simple example steps.
 
-* Configuration
+## Prerequisites
+  - Ruby
+  - Rails
+  - PostgreSQL
 
-* Database creation
+## Live Demo on Heroku
 
-* Database initialization
+  - [Railsbugdetplus](https://railsbudgetplus.herokuapp.com/)
 
```

Comment:
- [ ] Fix your heroku link

Reply:
Thank you, upon checking, the link is working. Maybe I am doing it in the wrong way? But when I click on the "Railsbudgetplus" link in the readme file, I am redirected to the Heroku app.